client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -115,18 +119,45 @@ class JetpackChecklist extends PureComponent< Props > {
 					completedTitle={ task.completedTitle }
 					description={ task.description }
 					duration={ task.duration }
+					firstIncomplete={ this.getFirstIncompleteTask( taskIds ) }
 					href={ task.getUrl( this.props.siteSlug, isComplete ) }
+					id={ taskId }
+					key={ taskId }
 					onClick={ this.handleTaskStart( {
 						taskId,
 						tourId: get( task, [ 'tourId' ] ),
 					} ) }
+					onTaskTitleClick={ this.onTaskTitleClick }
 					title={ task.title }
-					key={ taskId }
+					selectedTaskId={ this.state.selectedTaskId }
```

Comment:
From an API design POV, a component named `Task` -- used beside many other `Task` components as children of a `Checklist` -- shouldn't have a prop named `selectedTaskId`. The `Task` component doesn't care about task IDs -- all it cares about is whether it's selected or not.

A better choice is thus a `selected` (or maybe `expanded`) boolean prop.